yb-voyager/src/srcdb/postgres.go

```diff
@@ -165,7 +165,7 @@ func (pg *PostgreSQL) getConnectionUri() string {
 }
 
 func (pg *PostgreSQL) ExportSchema(exportDir string) {
-	pgdumpExtractSchema(pg.source.Schema, pg.getConnectionUri(), exportDir)
+	pgdumpExtractSchema(pg.source, pg.source.Schema, pg.getConnectionUri(), exportDir)
```

Comment:
since you are passing complete `pg.source` struct as an argument to the function, then `pg.source.Schema` is not required